pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -3609,6 +3611,68 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			expectMatchingPodCreation(vmi)
 		})
 	})
+
+	Context("Event handling", func() {
+		var (
+			vmi *virtv1.VirtualMachineInstance
+		)
+
+		indexVirtualMachine := func(vmi *virtv1.VirtualMachineInstance) {
+			Expect(controller.vmiIndexer.Add(vmi)).To(Succeed())
+			_, err := virtClientset.KubevirtV1().VirtualMachineInstances(vmi.Namespace).Create(context.Background(), vmi, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+		}
+
+		pvcWith := func(requestSize, capacity, rv string) *k8sv1.PersistentVolumeClaim {
+			parsedReq := resource.MustParse(requestSize)
+			parsedCap := resource.MustParse(capacity)
```

Comment:
You can inline them

Reply:
I prefer declaring these variables here, in my opinion it's more clear to pass the arguments as strings instead of pasing the resource everytime.

Reply:
The resources are parsed every time this function is called. I meant you could inline these two vars into L3627 and L3630.

Reply:
ah okay I thought you mean to pass the already parsed argument to the function. I'll change it.

Reply:
ack